src/flow/darcy_flow_mh.cc

```diff
@@ -929,8 +929,11 @@ void P1_CouplingAssembler::assembly(LinSys &ls) {
         // base fce = a0 + a1*t0 + a2*t1
         //         a0     a1      a2
         base_2D << 1.0 << 0.0 << -2.0 << arma::endr //point on side 0
```

Comment:
I don't know where this matrix is coming from, but it should be made independent of ref. element numbering.